Small Changes

Haeri opened the pull request:
- Added 404 Page (Closes https://github.com/mp-access/Backend/issues/340)
- Added course and owner infos to course display
- Changed "Close" to a more appropriate term (Closes https://github.com/mp-access/Backend/issues/328)

Haeri committed:
Small Changes

- Added 404 Paeg
- Added course and owner infos to course display
- Changed "Close" to a more appropriate term

a-a-hofmann commented:
Nice 👍 
Would be nice if we could redirect to the 404 page also if we get a 404 from the backend
e.g. a request to http://localhost:3000/courses/x returns a 404 because course x does not exist -> redirect to 404

Haeri commented:
> Nice 
> Would be nice if we could redirect to the 404 page also if we get a 404 from the backend
> e.g. a request to http://localhost:3000/courses/x returns a 404 because course x does not exist -> redirect to 404

Ah yeah that's a good idea! But I'm not sure how I would go about that. The request happens in CourseService.js. I would have to change the catch function. But I'm not sure how I can render a 404 page without redirecting to it.

Haeri committed:
404 Redirect

- Added 404 redirect for wrong exercise, assignment and exercie ids
- Removed the error redirect, so we can F5
- Removed Code

Haeri committed:
Merge branch 'dev' into feature/small-changes
